src/imitation/scripts/eval_policy.py

```diff
@@ -105,13 +104,15 @@ def eval_policy(
         )
 
     if rollout_save_path:
-        types.save(rollout_save_path.replace("{log_dir}", log_dir), trajs)
+
+        types.save(log_dir / rollout_save_path.lstrip("{log_dir}/"), trajs)
```

Comment:
`lstrip()` strips *characters* not substring replacement, I don't think this is what you want:

```python
>>> rollout_save_path='{log_dir}/{log}/{dir}/log/dir/dddrrriiibar'
>>> rollout_save_path.lstrip("{log_dir}/")
'bar'
```

Reply:
Okay yes this is terrible, thanks for pointing it out, I thought it literally matched the entire string.

Reply:
Weird that this didnt make any test fail.